packages/@aws-cdk/aws-dynamodb/test/test.dynamodb.ts

```diff
@@ -1333,6 +1333,123 @@ export = {
       test.done();
     }
   },
+
+  'import': {
+    'static import(ref) allows importing an external/existing table from arn'(test: Test) {
+      const stack = new Stack();
+
+      const tableArn = 'arn:aws:dynamodb:::table/MyTable';
```

Comment:
This is wrong :) Table ARNs contain account and regions: https://docs.aws.amazon.com/IAM/latest/UserGuide/list_amazondynamodb.html#amazondynamodb-resources-for-iam-policies

Reply:
Will fix it.